src/compiler/emitter.ts

```diff
@@ -2150,9 +2150,9 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
             }
 
             // Return true if identifier resolves to an exported member of a namespace
-            function isNamespaceExportReference(node: Identifier) {
+            function isExportReference(node: Identifier) {
                 const container = resolver.getReferencedExportContainer(node);
-                return container && container.kind !== SyntaxKind.SourceFile;
+                return !!container;

```

Comment:
If you had a namespace, then the container is probably a namespace (i.e. `ModuleDeclaration` or `ModuleBlock`).

Are there any other containers I need to be aware of? Potentially statics on a class?